hyperspy/gui/egerton_quantification.py

```diff
@@ -369,10 +369,12 @@ def _interpolator_kind_changed(self, old, new):
         self.span_selector_changed()
 
     def _ss_left_value_changed(self, old, new):
-        self.span_selector_changed()
+        if not np.isnan(self.ss_right_value):
+            self.span_selector_changed()
 
     def _ss_right_value_changed(self, old, new):
-        self.span_selector_changed()
+        if not np.isnan(self.ss_right_value):
```

Comment:
Sorry for the noise again: this doesn't cause any troubles in the
present case because we don't actually reset the span selector. But
again, it's better to check as you propose. (If I'm still getting
something wrong please let me know.)
On Tue, 2015-07-28 at 07:43 -0700, vidartf wrote:
> if not (np.isnan(self.ss_left_value) or
> np.isnan(self.ss_right_value)):